[2020-11-04T21:09:41.743] [ERROR] errorLogger - 
*************** error log start ***************
request method: POST
request originalUrl:  /api/server/serverCreate
request client ip:  ::ffff:127.0.0.1
request body: 
{"serverid":"","servername":"112","plaform":["1"],"channel":["测试渠道"],"ip":"","display":"1","srttime":"2020-11-26T16:00:00.000Z","address":"111123123","test":"0","page":1,"pagesize":20,"gameid":"13"}
response time: 1604495381741
err name: undefined
err message: 参数不合法
err stack: undefined
*************** error log end ***************

[2020-11-04T21:18:33.041] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/mail/stopMailSend?id=64
request client ip:  ::ffff:127.0.0.1
request query:  {"id":"64","gameid":"1"}
response time: 1604495913041
err name: SequelizeDatabaseError
err message: relation "set" does not exist
err stack: SequelizeDatabaseError: relation "set" does not exist
    at Query.formatError (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at MailService._callee11$ (H:\致命代码\fancyguo-gm\server\src\app\service\/mail.js:245:32)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
*************** error log end ***************

